examples/me_arm_example.py

```diff
@@ -0,0 +1,95 @@
+import functools
```

Comment:
What do you think about keeping notebooks for the examples?

Reply:
The issue we have with notebooks is that they are not super compliant with the way we (and other people) start new projects.
An example file makes it possible for newcomers (and also for us) to start new projects easily by:

 1. Copying the example file
 2. And modifying the copy by adding/changing the elements present in the file.